### ISSUE TITLE
/events API Endpoint Should Separate JsonObjects

### ISSUE THREAD

**Issue by amihaiemil** [moby/moby#40840]
I originally opened [this Issue](https://github.com/docker/for-mac/issues/4474) and was told to move it in this repo.

  - [X] I have tried with the latest version of my channel (Stable or Edge)

### Expected behavior
Docker Server should send some separator between the ``/events`` JSON Objects
### Actual behavior
Docker Server sends the JSONs one after another. The following are 2 docker ``pull`` events, which were sent by the server one after another without any separator (newline added by me for readability):

```json
{"status":"pull","id":"ubuntu:latest","Type":"image","Action":"pull","Actor":{"ID":"ubuntu:latest","Attributes":{"name":"ubuntu"}},"scope":"local","time":1587233840,"timeNano":1587233840345201553}

{"status":"pull","id":"hello-world:latest","Type":"image","Action":"pull","Actor":{"ID":"hello-world:latest","Attributes":{"name":"hello-world"}},"scope":"local","time":1587233845,"timeNano":1587233845411445535}
```
### Information

  - macOS Version: ``10.14.6``
  - Docker for Mac, Engine ``19.03.8``, API version ``1.40``.

### Steps to reproduce the behavior
1. Start Docker, then run the following CURL command in a terminal:

```shell
curl --no-buffer -XGET --unix-socket /var/run/docker.sock http://v1.40/events
```

2. Open another terminal and run some commands that generate events. For instance, the following two commands:

```shell
docker pull hello-world
docker pull ubuntu
```

3. Back in the first terminal, you will see that CURL has printed the received JSON objects without any separator between them.

A separator between the sent objects would be beneficial for the clients. I personally have a Java Client and I am basically parsing the incoming InputStream blindly, because I can't really know where one packet ends and where the next one begins.

**Comment by cpuguy83** [comment]
I'm not sure what adding a delimter (e.g. a newline) to the stream would do for all clients involved.

Have you looked into use a stream decoder?

**Comment by amihaiemil** [comment]
@cpuguy83 I'm not sure what stream decoding technique to use. I simply read bytes in a ``byte[]`` from the InputStream and try to parse what I got: https://github.com/amihaiemil/docker-java-api/blob/master/src/main/java/com/amihaiemil/docker/RtEvents.java#L194

If the read ``byte[]`` constitutes a JsonObject, parsing will work, otherwise not. But it's still kind of a heuristic, maybe Docker sends me 1 Json and half of the second one in that byte buffer I allocated.

I'm not sure how to go about this. A delimiter would probably help. Or at least a guarantee that Docker will send only one Json at a time?
